src/renderer/assets/files/dd-template.yaml

```diff
@@ -0,0 +1,17 @@
+0x387aed21a43ed189c8f0d14aab6f05f715f1adfe670efa6090208ddd4efe991f: #vault public address
```

Comment:
@egor0798 please update according the https://github.com/novasamatech/research-docs/pull/58/files#diff-3d11decbee0a3ce1d535b583bc722d7c67d7d99f9cb0864666af74eea770c954